lib/renderer.js

```diff
@@ -1164,10 +1183,12 @@
         // 4. The "include-ancestor-masks" config option is NOT set
         // 5. None of the layers has zero bounds. Sometimes they aren't computed and set to all 0's
         // 6. The layer is clipped, in which case layer.bounds is the clipped size and not the layer size
-        var hasComplexTransform = (component.hasOwnProperty("scale") && component.scale % 1 !== 0) ||
-                component.hasOwnProperty("width") || component.hasOwnProperty("height"),
+        var layer = component.layer,
+            hasComplexTransform = layer &&
```

Comment:
I see.  You may be right that it is a limitation with `_getSettingsWithApproximateBounds `, but specifically it is that in that method there is no consideration of an explicit height/width, only providing for scalar values.  see [here](https://github.com/adobe-photoshop/generator-assets/blob/ea884529c40f5b3752273fad4a88be17f0a9b207/lib/renderer.js#L892-L894) vs the bit in exact bounds [here](https://github.com/adobe-photoshop/generator-assets/blob/ea884529c40f5b3752273fad4a88be17f0a9b207/lib/renderer.js#L800-L801)

So, I think your point is that I am missing an opportunity to optimize for layer comps by updating `_getSettingsWithApproximateBounds` to handle height/width calculations based on the less expensive data.  My plea for my newest PR (393) is that we are just reverting back to the functionality before this PR(374), and consider this optimization in the future.

To answer your question, the badness I'm seeing now is that layer comps which are scaled (via the image assets plugin) with an explicit height/width (such as `200x200 comp1.png`) are getting output *at original dimension*.